src/components/OSCALControlProse.js

```diff
@@ -168,7 +169,8 @@ function getTextSegment(text, key) {
   }
   return (
     <Typography component="span" key={key}>
-      {text}
+      {/* {text} */}
+      <OSCALMarkdownProse text={text} />
```

Comment:
Why not just replace the `Typography` with an `OSCALMarkdownProse`?

Reply:
<img src ="https://user-images.githubusercontent.com/106985986/183452922-6ba532aa-7b17-4d58-adfb-431d6dc5bdda.png" width=400px><img src = "https://user-images.githubusercontent.com/106985986/183453120-7a986e22-3be1-44f3-a2ab-125929b84dbc.png" width = 350px>

The code above resulted in the output on the right
I also tried this one
<img src="https://user-images.githubusercontent.com/106985986/183454160-2c8a2655-0be9-4953-982f-c84e95f3b9d8.png" width = 400px><img src = "https://user-images.githubusercontent.com/106985986/183454305-247d9b70-533d-4976-9a45-b24cff43b81c.png" width = 400px>
(The output for the second one is the same as the one above)

Would the next approach be to look at `react-markdown` and adjust the settings with the new line or is there another way to change the `getTextSegment()` function

Reply:
Wrapping an `OSCALMarkdownProse` component also gives the same output. I believe it is because `react-markdown` adds a new line at the end of their component so there will be a new line at the end of each span which knocks down text to the next line.
<img src = "https://user-images.githubusercontent.com/106985986/183457496-70db5026-f7f5-43f9-97df-33d689aba91d.png">

Reply:
See https://github.com/remarkjs/react-markdown/issues/42